addons/events/fnc_addPlayerEventHandler.sqf

```diff
@@ -121,115 +140,134 @@ if (_id != -1) then {
         GVAR(oldGroup) = grpNull;
         GVAR(oldLeader) = objNull;
         GVAR(oldWeapon) = "";
+        GVAR(oldMuzzle) = [objNull, "", ""];
+        GVAR(oldWeaponMode) = [objNull, GVAR(oldMuzzle), ""];
         GVAR(oldLoadout) = [];
         GVAR(oldLoadoutNoAmmo) = [];
         GVAR(oldVehicle) = objNull;
-        GVAR(inVehicle) = false;
         GVAR(oldTurret) = [];
         GVAR(oldVisionMode) = -1;
         GVAR(oldCameraView) = "";
         GVAR(oldFeatureCamera) = "";
         GVAR(oldVisibleMap) = false;
-        GVAR(oldUAVControl) = [];
-        GVAR(controlledEntity) = objNull;
+        GVAR(oldState) = [];
 
         GVAR(playerEHInfo) pushBack addMissionEventHandler ["EachFrame", {call FUNC(playerEH_EachFrame)}];
         [QFUNC(playerEH_EachFrame), {
```

Comment:
Can we maybe just move this to seperate file?
Makes this file be quite overloaded with stuff that doesn't _really_ belong